app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/ui/tracker_activity/DeviceShieldTrackerActivityViewModel.kt

```diff
@@ -252,6 +269,10 @@ class DeviceShieldTrackerActivityViewModel @Inject constructor(
         object OpenVpnSettings : Command()
         object ShowAppTpEnabledCta : Command()
     }
+
+    companion object {
+        const val WINDOW_INTERVAL_HOURS = 24L
```

Comment:
from the comment above, it would also remove this from the `VM` and push it to the `Store`